### ISSUE TITLE
Situationtriggers aren't reflected correctly in the API

### ISSUE THREAD

**Issue by LaviniaStiliadou** [OpenTOSCA/container#132]
Hello everyone,

I am currently integrating situations and situationtriggers into the OpenTOSCA UI as part of my bachelor thesis. Unfortunately, the following problems sometimes occur when creating and managing situation triggers:

1) The situations of situation triggers sometimes disappear. 
2) Input parameters  disappear after the POST request. 
3) The CsarID disappears after the POST request.

**Steps to Reproduce:**
```xml
<?xml version="1.0" encoding="UTF-8"?>
<Situation>
    <ThingId>Kalle</ThingId>
    <SituationTemplateId>AtHome</SituationTemplateId>
    <Active>false</Active>
</Situation>
```
```xml
<?xml version="1.0" encoding="UTF-8"?>
<SituationTrigger>
    <Situations><SituationId>74</SituationId></Situations>
    <CsarId>MyTinyToDo_Bare_Docker.csar</CsarId>
<onActivation>true</onActivation>
<isSingleInstance>true</isSingleInstance>
<InterfaceName>OpenTOSCA-Lifecycle-Interface</InterfaceName>
<OperationName>initiate</OperationName>
<InputParameters>
    <InputParameter>
        <name>ApplicationPort</name>
        <Value>9990</Value>
        <Type>String</Type>
    </InputParameter>
    <InputParameter>
        <name>ContainerSSHPort</name>
        <Value>9991</Value>
        <Type>String</Type>
    </InputParameter>
    <InputParameter>
        <name>DockerEngineURL</name>
        <Value>tcp://dind:2375</Value>
        <Type>String</Type>
    </InputParameter>
</InputParameters>
</SituationTrigger>
```

After GET Request:
```json
{
    "id": 343,
    "situation_ids": [
        74
    ],
    "on_activation": true,
    "interface_name": "OpenTOSCA-Lifecycle-Interface",
    "operation_name": "initiate",
    "input_params": [],
    "event_probability": -1.0,
    "single_instance": false,
    "_links"
}
```
Thank you in advance.

**Comment by nyuuyn** [comment]
Bug was fixed. Don't forget to pull the newest images when they are available at dockerhub :)

**Comment by nyuuyn** [comment]
FIx/CHeck (as we had a mega merge) the missing CSAR id

**Comment by nyuuyn** [comment]
fixed in e45f101148e6d2b2886162d8b74db02dc02aba6a
